app.py

```diff
@@ -133,11 +133,11 @@ def predict(uuid):
 		query = pd.DataFrame([json_])
 		return jsonify({'prediction': list(model.predict(query))})
 
-	except Exception, e:
-		print 'error'
-		print e
+	except getopt.GetoptError as e:
```

Comment:
So for generic exception handling in Python 3 we can do `except:` or `except Exception as e:`. However this is not [encouraged](http://stackoverflow.com/questions/14797375/should-i-always-specify-an-exception-type-in-except-statements) and it's better to handle specific exceptions. This was bad design on my part being lazy and capturing all exceptions generically. It's probably better to capture each exception separately. For now feel free to just use this generic handler pattern:
```
try:
    ...
except Exception as e:
    ...
```

I will fix this later.

A few examples: https://gist.github.com/amirziai/93dd997756b2ddb7d5430e33d39558b4

I'll add some tests and hook up travis to get things under control.